src/App.tsx

```diff
@@ -1,61 +1,116 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUserById = (userId: number): User | null => (
+  usersFromServer.find(user => user.id === userId) || null
+);
+
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.id),
```

Comment:
The `user` property should be assigned using `getUserById(todo.userId)` instead of `getUserById(todo.id)`. This ensures that the correct user is associated with each todo item.